index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <title>Fork Me! FCC: Test Suite Template</title>
+        <link rel="stylesheet" href="css/style.css" />
+    </head>
+
+    <body>
+        <div>“Hello World, we are software developers!”</div>
```

Comment:
- [x] Try to use semantic `HTML` tags whenever is possible.
- suggestion: For this case, you can use the `<h1> </h1>` tag, as this is the first header of your page.